### PR TITLE
Extended SpamFilteringEnabledType Description

### PR DESCRIPTION
**Pull request by YuriySamorodov**
Added extended description for SpamFilteringEnabledType parameter

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -414,13 +414,12 @@ Accept wildcard characters: False
...
-Determines if Spam filtering is enabled.
+Determines Spam filtering mode.
 
 Possible values:
-- Enabled
-- Disabled
-- EnabledWithoutIVR (Choosing this value will disable the default spam checking which integrates the Captcha IVR Bot into the flow. The call will not be redirected to the bot for checking against fraudulent calls but will be allowed to go to the original target if the spam score is not high.)
-
+- Enabled - Spam Filtering is fully enabled. Both Basic and Captcha IVR checks are performed. In case the call is considered as spam, user will get "Spam Likely" notification in Teams
+- Disabled - Spam Filtering is completely disabled. No checked are performed. "Spam Likely" notification will not appear.
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:421]:
```suggestion
- Disabled - Spam Filtering is completely disabled. No checks are performed. "Spam Likely" notification will not appear.
```

**Review by get-itips — COMMENTED** [review]
Hello @YuriySamorodov thanks for your contribution, as you might know, this is somewhat difficult to test, can you tell us how you were able to test this? Thank you

**Comment by YuriySamorodov** [comment]
> Hello @YuriySamorodov thanks for your contribution, as you might know, this is somewhat difficult to test, can you tell us how you were able to test this? Thank you

These details have been provided by Microsoft Support (Ticket 27190817) and @dinabogdan

**Comment by dariomws** [comment]
@dinabogdan, Could you kindly confirm this information? Thanks!

**Comment by dariomws** [comment]
@dinabogdan, Could you kindly confirm this information? Thanks!

**Comment by yogkumgit** [comment]
@tiburd Kindly take a suitable action. Thanks

**Comment by dinabogdan** [comment]
Hi! Sorry for so delayed response. Indeed, I have discussed with @YuriySamorodov offline :) Thanks!

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**